megamek/src/megamek/common/verifier/TestMek.java

```diff
@@ -786,21 +786,16 @@ public String getName() {
         return "Mek: " + mek.getDisplayName();
     }
 
-    /**
-     * calculates the total weight of all armored components.
-     */
     @Override
     public double getArmoredComponentWeight() {
         double weight = 0.0;
         double cockpitWeight = 0.0;
-        for (int location = Mek.LOC_HEAD; location < mek.locations(); location++) {
+        for (int location = 0; location < mek.locations(); location++) {
```

Comment:
Why the change from an ENUM to an Integer?

Reply:
It was not an enum, just a number. In this spot, the LOC_HEAD is meaningless, it actually wants to cycle from 0 to the number of locations. Having LOC_HEAD there is not in any way more informational, it's nonsense.

Reply:
Put differently: I have to hover the LOC_HEAD to see that it's 0 and this actually cycles all locations. LOC_HEAD could be 2, then this for() would be special. But it isnt special, and there is no reason not to see that immediately.

Reply:
Ahh. So it was a static value. Was just curious. These questions aren't meant to stop the merge. Just curiosity.